EdFi.Ods.Utilities.Migration/Providers/MigrationConfigurationProvider.cs

```diff
@@ -61,6 +61,8 @@ public MigrationConfigurationProvider(IOdsMigrationManagerResolver odsMigrationM
 
             RaiseErrorIfUpgradePathNotSupported(upgradeVersionConfiguration);
 
+            upgradeVersionConfiguration.FeaturesBeforeUpgrade = autoDetectedVersion?.ExistingFeatures?.ToList();
```

Comment:
As discussed, this still needs to coalesce to an empty list, not set to null.

Reply:
corrected